rholang/src/test/scala/coop/rchain/rholang/interpreter/matcher/StreamTSpec.scala

```diff
@@ -80,6 +80,19 @@ class StreamTSpec extends FlatSpec with Matchers {
 
     assert(StreamT.run(combined).value() == (reference ++ reference))
   }
+
+  it must "be stacksafe and lazy for a stacksafe F when calling dropTail" in {
+    type StreamTCoeval[A] = StreamT[Coeval, A]
+
+    val reference                    = Stream(1)
+    val huge: StreamTCoeval[Int]     = StreamT.fromStream(Coeval.now(reference))
+    val throwing: StreamTCoeval[Int] = StreamT.liftF[Coeval, Int](Coeval.delay(???))
```

Comment:
Seeing an undefined value in a test is new to me. Why not make `???` a concrete value, even if it's an arbitrary one?

Reply:
This is specifically so that an exception is thrown if that value is accessed :) Hence the name - `throwing`.